dist/infiniband-radar-daemon.spec

```diff
@@ -0,0 +1,67 @@
+# run "spectool -g -R infiniband-radar-daemon.spec" to automatically download 
+# source files. spectool is part of the rpmdevtools package
+
+%define use_systemd (0%{?fedora} && 0%{?fedora} >= 18) || (0%{?rhel} && 0%{?rhel} >= 7) || (0%{?suse_version} && 0%{?suse_version} >=1210)
+
+Name:		infiniband-radar-daemon
+Version:	2.07
+Release:	1%{?dist}
+Summary:	InfiniBand Radar - Service Daemon
+
+Group:		System Environment/Base
+License:	GPLv3
+URL:		https://github.com/infiniband-radar/%{name}
+Source0:	https://github.com/infiniband-radar/%{name}/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:	cmake3 gcc-c++ libibverbs-devel libibmad-devel infiniband-diags-devel libcurl-devel
+%{?systemd_requires}
+BuildRequires: systemd
+Requires:	libibverbs libibmad
```

Comment:
I am also using libcurl in order to communicate with the api server.

Reply:
True, I forgot that, my bad.
I also added infiniband-diags that was missing from the runtime dependencies (for libibnetdisc.so), and remove libibverbs that doesn't seem to be used to either build nor run.